### ISSUE TITLE
talend aatams_acoustic_reporting failing

### ISSUE THREAD

**Issue by danfruehauf** [aodn/content#79]
Been like that for the last 12 days.

This is a small digest from the log:
```
Exception in component iPostgresqlOutput_1
java.sql.BatchUpdateException: Batch entry 9,760 INSERT INTO "dw_aatams_acoustic"."aatams_acoustic_detections_data" ("detection_id","project_name","installation_name","station_name","bottom_depth","common_name","transmitter_id","release_date","release_locality","detection_date","longitude","latitude","sex","measurement_type","measurement_value","measurement_unit","phylum","order_name","scientific_name","geom","detected") VALUES (82014474,Logan River Catadromous Fish Migration,Upper Logan River,TBLJA,2.3,Australian Bass,A69-1601-9054,2013-04-05 02:00:00.000000 +1100,Teviot Brook,2013-04-14 14:25:08.000000 +1000,152.93,-27.83,UNKNOWN,WEIGHT,310.0,g,Chordata,Perciformes,Macquaria novemaculeata,SRID=4326;POINT(152.93 -27.83),1) was aborted.  Call getNextException to see the cause.
        at org.postgresql.jdbc2.AbstractJdbc2Statement$BatchResultHandler.handleError(AbstractJdbc2Statement.java:2512)
        at org.postgresql.core.v3.QueryExecutorImpl$ErrorTrackingResultHandler.handleError(QueryExecutorImpl.java:279)
        at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1310)
        at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:328)
        at org.postgresql.jdbc2.AbstractJdbc2Statement.executeBatch(AbstractJdbc2Statement.java:2574)
        at aatams_acoustic_reporting.populate_layers_0_1.Populate_layers.tPostgresqlInput_1Process(Populate_layers.java:1915)
        at aatams_acoustic_reporting.populate_layers_0_1.Populate_layers.tPostgresqlConnection_1Process(Populate_layers.java:820)
        at aatams_acoustic_reporting.populate_layers_0_1.Populate_layers.iIncludeSdiLibraries_1Process(Populate_layers.java:697)
        at aatams_acoustic_reporting.populate_layers_0_1.Populate_layers.runJobInTOS(Populate_layers.java:3685)
        at aatams_acoustic_reporting.populate_layers_0_1.Populate_layers.runJob(Populate_layers.java:3552)
        at aatams_acoustic_reporting.aatams_acoustic_harvester_0_1.aatams_acoustic_harvester.tRunJob_4Process(aatams_acoustic_harvester.java:1841)
        at aatams_acoustic_reporting.aatams_acoustic_harvester_0_1.aatams_acoustic_harvester.tRunJob_1Process(aatams_acoustic_harvester.java:1728)
        at aatams_acoustic_reporting.aatams_acoustic_harvester_0_1.aatams_acoustic_harvester.iPostgresqlDbUpdate_1Process(aatams_acoustic_harvester.java:1563)
        at aatams_acoustic_reporting.aatams_acoustic_harvester_0_1.aatams_acoustic_harvester.runJobInTOS(aatams_acoustic_harvester.java:3783)
        at aatams_acoustic_reporting.aatams_acoustic_harvester_0_1.aatams_acoustic_harvester.main(aatams_acoustic_harvester.java:3603)
Exception in component tRunJob_4
java.lang.RuntimeException: Child job running failed
        at aatams_acoustic_reporting.aatams_acoustic_harvester_0_1.aatams_acoustic_harvester.tRunJob_4Process(aatams_acoustic_harvester.java:1852)
        at aatams_acoustic_reporting.aatams_acoustic_harvester_0_1.aatams_acoustic_harvester.tRunJob_1Process(aatams_acoustic_harvester.java:1728)
        at aatams_acoustic_reporting.aatams_acoustic_harvester_0_1.aatams_acoustic_harvester.iPostgresqlDbUpdate_1Process(aatams_acoustic_harvester.java:1563)
        at aatams_acoustic_reporting.aatams_acoustic_harvester_0_1.aatams_acoustic_harvester.runJobInTOS(aatams_acoustic_harvester.java:3783)
        at aatams_acoustic_reporting.aatams_acoustic_harvester_0_1.aatams_acoustic_harvester.main(aatams_acoustic_harvester.java:3603)
```

**Comment by xhoenner** [comment]
@danfruehauf I noticed that, this is due to an error in the metadata that someone put in. There's a PR waiting to be reviewed (https://github.com/aodn/harvesters/pull/165) that will sort out that issue.

**Comment by danfruehauf** [comment]
@xhoenner All good. Lets just keep track on that...

**Comment by jachope** [comment]
Hi Jac,

yes it can be closed, the harvester’s been re-deployed and runs successfully now.

Cheers,

Xavier

On 4 Mar 2015, at 3:45 pm, Jacqui Hope <Jacqui.Hope@utas.edu.au> wrote:

Hi Xavier,
 
Just doing a bit of a review on the content repo and I note that we may be able to close -https://github.com/aodn/content/issues/79 ??
 
Looks like the issue was fixed with aodn/harvesters#165
 
Thanks
Jac
